appeals/web/src/server/app/auth/auth.pipes.js

```diff
@@ -11,3 +14,25 @@ export const clearAuthenticationData = ({ session }, _, next) => {
 	authSession.destroyAuthenticationData(session);
 	next();
 };
+
+/** @typedef {Record<('setSiteVisit'|'setAppealStatus'|'setAppellantCaseStatus'|'setLpaQStatus'|'setAppealCaseData'|'setAppellantCaseData'|'setLpaQCaseData'), boolean>} CurrentPermissionSet */
+
+/**
+ * @param {string[]} currentUserGroups
+ * @returns {CurrentPermissionSet}
+ */
+const permissions = (currentUserGroups) => {
+	return {
+		setSiteVisit: currentUserGroups.indexOf(config.referenceData.appeals.inspectorGroupId) >= 0,
+		setAppealStatus:
+			currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0,
+		setAppellantCaseStatus:
+			currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0,
+		setLpaQStatus: currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0,
+		setAppealCaseData:
+			currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0,
+		setAppellantCaseData:
+			currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0,
+		setLpaQCaseData: currentUserGroups.indexOf(config.referenceData.appeals.caseOfficerGroupId) >= 0
+	};
```

Comment:
Using `.includes()` rather than `.indexOf()` might make this list more readable, especially if it gets longer in time.

For example:

setSiteVisit: currentUserGroups.includes(config.referenceData.appeals.inspectorGroupId),